src/components/KonnectorTile.jsx

```diff
@@ -23,14 +23,24 @@ const getKonnectorError = ({ error, t }) => {
     : null
 }
 
-const getErrorClass = ({ accountsCount, error, hide, userError }) => {
+const getErrorClass = ({
+  accountsCount,
+  error,
+  hide,
+  userError,
+  hasUpdate
+}) => {
   if (hide) return ''
 
   // userError must be checked first as it is also an error.
   if (userError) {
     return 'konnector-error konnector-error--connection'
   }
 
+  if (hasUpdate) {
+    return 'konnector-error--update'
```

Comment:
Other types of error have the class `konnector-error` in addition to the specific error class. Is there a reason why this error is not here ?

Reply:
Yes, we won't to have the full error displayed here (the red border with the breaking draw)